WoWsShipBuilder.Core/DataContainers/Ship/ManeuverabilityDataContainer.cs

```diff
@@ -14,6 +14,9 @@ public partial record ManeuverabilityDataContainer : DataContainerBase
     [DataElementType(DataElementTypes.KeyValueUnit, UnitKey = "Knots")]
     public decimal ManeuverabilityMaxSpeed { get; set; }
 
+    [DataElementType(DataElementTypes.KeyValueUnit, UnitKey = "Knots", NameLocalizationKey = "MaxReverseSpeed")]
```

Comment:
The naming in this class needs to be fixed (the maneuverability prefix should be dropped) but this is not something you should do. Just leaving this comment to have a reminder for later.